### PR TITLE
feat(wallet): Add pipelines for `payment-wallet-event-dispatcher`

### PR DESCRIPTION
**Pull request by giovanniberti**
### List of changes

 * add pipelines for `payment-wallet-event-dispatcher`

### Motivation and context

These pipelines are needed for the new [`pagopa-payment-wallet-event-dispatcher-service`](https://github.com/pagopa/pagopa-payment-wallet-event-dispatcher-service) microservice

### Type of changes

- [x] Add new pipeline
- [ ] Update pipeline configuration
- [ ] Remove pipeline

### :warning: If it's new pipeline with code review have you added pagopa-github-bot -> Role: admin?

- [x] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: azure-devops/wallet/06_pagopa-wallet-event-dispatcher-service.tf [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    uat_container_namespace = "pagopaucommonacr.azurecr.io"
+    # prod_container_namespace = "pagopapcommonacr.azurecr.io"
+
+  }
+  # deploy secrets
+  pagopa-wallet-event-dispatcher-service-variables_secret_deploy = {
+    git_mail                        = module.secrets.values["azure-devops-github-EMAIL"].value
+    git_username                    = module.secrets.values["azure-devops-github-USERNAME"].value
+    tenant_id                       = data.azurerm_client_config.current.tenant_id
+    wallet_token_test_dev           = module.wallet_dev_secrets.values["wallet-token-test-key"].value
```
Comment (infantesimone) [inline azure-devops/wallet/06_pagopa-wallet-event-dispatcher-service.tf:71]: are these tokens needed?

**Review by infantesimone — CHANGES_REQUESTED** [review]

File: azure-devops/wallet/06_pagopa-wallet-event-dispatcher-service.tf (hunk 1 above)
Comment (giovanniberti) [reply azure-devops/wallet/06_pagopa-wallet-event-dispatcher-service.tf:71]: Nope, removing

Comment (giovanniberti) [reply azure-devops/wallet/06_pagopa-wallet-event-dispatcher-service.tf]: Removed in 3c3f817

**Review by infantesimone — APPROVED** [review]

**Comment by infantesimone** [comment]
@diegolagospagopa There are some errors in the terraform plan regarding service connections of wallet domain, unrelated to this PR.
service connections changed?

**Review by infantesimone — APPROVED** [review]

**Pull request merged**